examples/basic/README.md

```diff
@@ -4,20 +4,20 @@ This example shows how a basic example to use JuiceFS in Kubernetes pod.
 
 ## Prerequisite
 
-Create JuiceFS filesystem in [JuiceFS web console](https://juicefs.com/console)
+Create JuiceFS filesystem using [juicefs format](https://github.com/juicedata/juicefs/#format-a-volume). Take Amazon S3 as an example:
+
+```
+./juicefs format --storage=s3 --bucket=https://<BUCKET>.s3.<REGION>.amazonaws.com redis://user:password@<HOST>:6379/1 <NAME>
+```
+
+
 
 Then create secret for access in Kubernetes.
 
 ```sh
->> kubectl -n default create secret generic juicefs-secret --from-literal=name=$JFS_NAME --from-literal=token=$JFS_TOKEN --from-literal=accesskey=$JFS_ACCESSKEY --from-literal=secretkey=$JFS_SECRETKEY
+>> kubectl -n default create secret generic juicefs-secret --from-literal=metaurl=redis://user:password@<HOST>:6379/1
```

Comment:
Does it drop support for enterprise edition?

Reply:
The enterprise edition is also supported. The docs need to be updated later(WIP), and only for community JuiceFS, enterprise docs are placed on juicefs.com.